cpu/arm7_common/Makefile.features

```diff
@@ -1,2 +1,5 @@
 FEATURES_PROVIDED += cpp
 FEATURES_PROVIDED += periph_pm
+FEATURES_PROVIDED += arch_arm7
```

Comment:
Please keep alphabetical order, here and in other modified places below.

Reply:
good point

Reply:
fixed